### PR TITLE
[Oxfordshire] display asset layers on map

### PR DESCRIPTION
**Pull request by struan**
This adds asset layers to the map for OCC for:
 - drains
 - grit bins
 - streetlights
 - traffic lights
 - trees

It also adds a roads layer for access to the USRN.

It includes the asset id and USRN in the description sent over Open311.

[skip changelog]

### CODE REVIEW

File: web/cobrands/oxfordshire/assets.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+});
+
+var streetlight_select = $.extend({
+    label: "${UNITNO}",
+    fontColor: "#FFD800",
+    labelOutlineColor: "black",
+    labelOutlineWidth: 3,
+    labelYOffset: 65,
+    fontSize: '15px',
+    fontWeight: 'bold'
```
Comment (davea) [inline web/cobrands/oxfordshire/assets.js:51]: There's something about this which makes it quite hard to determine what's going on in busy areas:

image

I tried changing the font and outline sizes, but didn't make much difference. I think it's all the yellow. Could the asset markers perhaps be a different, more contrasting, colour? With a bigger font size too it pops a bit more:

image

Comment (zarino) [reply web/cobrands/oxfordshire/assets.js:51]: I agree with the idea of adding stronger colour contrast between the number and the background (which will often be covered with dots). Or even just generally between the "selected" asset marker, and the background markers.

Blue for the dots looks ok – although we’ll need to make it’s not too easily confused with the blue "here’s your location" dot that was recently added to FMS. If we change the colour, we’ll also need update the yellow dot in the sidebar "pick an asset from the map" hint, assuming that’s still there.

Comment (struan) [reply web/cobrands/oxfordshire/assets.js:51]: Spot the location marker:

Screenshot 2020-06-15 at 11 56 45

I will try some other colours.

Comment (struan) [reply web/cobrands/oxfordshire/assets.js:51]: The dark green from the header seems to be ok:

Screenshot 2020-06-15 at 12 01 23

Comment (struan) [reply web/cobrands/oxfordshire/assets.js:51]: and against some dots:

Screenshot 2020-06-15 at 12 02 06

**Comment by dracos** [comment]
Made a few small changes before merging to reviewed branch: used fixmystreet.staging to pick tilma host rather than URLs; set a default stylemap for Tree/Traffic lights so all consistent; renamed asset_* to owned_* for clarity; included hover on streetlight stylemap; renamed Pavement to Pavements to match category name.

**Comment by dracos** [comment]
More changes needed: couldn't select owned asset once non-owned asset selected; only request required data from WFS servers; owned code not running on grit bins; layer name fixed and attribute renamed.

**Pull request merged**